### ISSUE TITLE
function id signature set

### ISSUE THREAD

**Issue by williballenthin** [fireeye/capa#443]
which functions do we want signatures for? 

this enables:
  - avoid FP matches in these library functions
  - match against calls to these library functions

**Comment by williballenthin** [comment]
- [libcmt](https://github.com/Maktm/FLIRTDB/blob/1f5763535e02d7cccf2f90a96a8ebaa36e9b2495/cmt/windows/libcmt_15_msvc_x64.pat)
![image](https://user-images.githubusercontent.com/156560/109456833-e54f9800-7a16-11eb-9b02-66a097057a6c.png)
  - [ucrt](https://devblogs.microsoft.com/cppblog/introducing-the-universal-crt/)
![image](https://user-images.githubusercontent.com/156560/109456916-0adca180-7a17-11eb-8441-02e9ca1546b9.png)
![image](https://user-images.githubusercontent.com/156560/109456722-a7eb0a80-7a16-11eb-8599-1b90786af548.png)

**Comment by williballenthin** [comment]
ideally use a matrix of visual studio releases, too

**Comment by williballenthin** [comment]
popular open source libraries. which ones? capa identifies the following, which seems to suggest some prevalence in malware:

![image](https://user-images.githubusercontent.com/156560/109457264-c00f5980-7a17-11eb-92c9-66f889e2a470.png)

need to find a good source .lib archives.

**Comment by mr-tz** [comment]
boost (https://www.boost.org/) and other crypto and compression libraries come to mind

also at least run-time support for gcc (mingw) and clang

**Comment by williballenthin** [comment]
vcpkg makes this so easy!

```
vcpkg.exe install install --triplet x32-windows-static cryptopp curl detours openssl mbedtls zlib
vcpkg.exe install install --triplet x64-windows-static cryptopp curl detours openssl mbedtls zlib
```

**Comment by williballenthin** [comment]
i put all this in a [Windows docker container](https://github.com/williballenthin/siglib/blob/main/Dockerfile) (which took quite a bit of experimenting/learning) only to find that GH Actions doesn't support Windows containers.

so can build this stuff reproducibly on my computer. if its meaningful, we can probably deploy CI to appveyor and build on windows there.

**Comment by williballenthin** [comment]
- protobuf

**Comment by williballenthin** [comment]
this is closed in #446
